### DISCUSSION TITLE
Countdown disappears from my activity bar

### DISCUSSION THREAD

**Discussion by paivadaniel** [focustimerhq/FocusTimer#684] [Q&A]
I use Ubuntu 22.04 and the gnome shell extension to show the time countdown in the activity bar, however, whenever I restart the computer and start gnome pomodoro, the icon/count does not appear in the activity bar, to make it appear I must deactivate/activate the gnome shell extension, and I do this at the url: https://extensions.gnome.org/extension/53/pomodoro/

How to make the counter automatically shown in the activity bar whenever using gnome pomodore? I mean without having to restart the gnome shell extension.
​

**Comment by kamilprusko** [comment]
> whenever I restart the computer and start gnome pomodoro, the icon/count does not appear in the activity bar

Please use the [Extensions](https://flathub.org/en-GB/apps/org.gnome.Extensions) app - you should have it preinstalled. Check whether the extensions are disabled in general. The system may disable extensions in case of an error.

**Comment by paivadaniel** [comment]
> > whenever I restart the computer and start gnome pomodoro, the icon/count does not appear in the activity bar
> 
> Please use the [Extensions](https://flathub.org/en-GB/apps/org.gnome.Extensions) app - you should have it preinstalled. Check whether the extensions are disabled in general. The system may disable extensions in case of an error.

I installed Extensions as you instructed, and it apparently resolved the issue. I will use Pomodoro this week, and if the error occurs again, I will try to disable the other gnome extensions to see if the error persists, and if so, I will update this post with more information.
